src/main/java/seedu/address/model/person/Person.java

```diff
@@ -139,6 +139,15 @@ public boolean isSamePerson(Person otherPerson) {
                 && otherPerson.getPhone().equals(getPhone());
     }
 
+    /**
+     * Determines whether the person owes money.
+     *
+     * @return true if the person owes money
+     */
+    public boolean owesMoney() {
```

Comment:
Do you think which would be a better naming for the function? `owedMoney()` or `isOweingMoney()`?

Reply:
👍 Have changed it to isOweingMoney()